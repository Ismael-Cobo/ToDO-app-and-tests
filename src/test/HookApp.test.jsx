import { shallow } from "enzyme"
import { HookApp } from "../../HookApp"

describe('prueba para el componente <HookApp />', () => {

    test('debe de cargar bien el componente', () => {

        const wrapper = shallow(<HookApp />)

        expect(wrapper).toMatchSnapshot()
    })
})