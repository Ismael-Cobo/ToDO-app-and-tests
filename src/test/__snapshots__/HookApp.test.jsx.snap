// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba para el componente <HookApp /> debe de cargar bien el componente 1`] = `
<h1>
  HookApp
</h1>
`;
